tests/testthat/test-plot_with_settings_ui.R

```diff
@@ -7,3 +7,4 @@ testthat::test_that("plot_with_settings_ui returns shiny.tag.list", {
 })
 
 library(shinytest2)
```

Comment:
@asbates the point here was to simplify the tests by testing as much as possible on the server side. What UI tests are you missing exactly?

Reply:
Woops, this comment should have been the line with the stray `library` call.

Reply:
@mhallal1 One example is [showing/hiding](https://github.com/insightsengineering/teal.widgets/blob/26f392d0ef997d4da33c3c25b3a2157939494099/tests/testthat/apps/pws_base/tests/testthat/test-shinytest2.R#L54) of the table/plot. The specific test linked is for `plot_with_settings` but `table_with_settings` has a show/hide argument as well.

Reply:
OK so I missed [this test](https://github.com/insightsengineering/teal.widgets/blob/68fc3aa33339315432a272029dfac3a197174d12/tests/testthat/test-table_with_settings.R#L24) for show/hide. Doh! However, I don't think this is really testing if the table is hidden or not. It may pass but it's not really checking the visibility.